SPECS/linux/config

```diff
@@ -800,13 +800,8 @@ CONFIG_HAVE_ARCH_PREL32_RELOCATIONS=y
 #
 # CONFIG_GCOV_KERNEL is not set
 CONFIG_ARCH_HAS_GCOV_PROFILE_ALL=y
-CONFIG_PLUGIN_HOSTCC="g++"
+CONFIG_PLUGIN_HOSTCC=""
 CONFIG_HAVE_GCC_PLUGINS=y
-CONFIG_GCC_PLUGINS=y
```

Comment:
You will need to leave these as is, to avoid breaking other features in Photon. (I believe we have a GCC  plugin patch that needs this config to remain enabled).